source/beamline/AMEnumeratedControl.h

```diff
@@ -11,6 +11,9 @@ class AMEnumeratedControl : public AMPseudoMotorControl
     Q_OBJECT
 
 public:
+	/// Enumeration of some default option indices.
+	enum DefaultOptions { Unknown = -1, Invalid = -2 };
```

Comment:
Nope, they aren't used anywhere yet. I tried to implement them early in this pull request, but I'm not ready for them yet. The plan is to use them eventually. Should I take them out?